tools/loris_db_upgrade.php

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/php
+<?php
+
+/**
+ * This script will upgrade the loris database to the lastest. 
+ * Usage: php loris_db_upgrade.php [-D]
+ *
+ * PHP Version 7
+ *
+ * @category Install
+ * @package  Loris
+ * @author   Shen Wang <shen.wang2@mcgill.ca>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ * @package behavioural
+ */
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once __DIR__ . "/generic_includes.php";
+ set_include_path(get_include_path().":../project/libraries:../php/libraries:");
+require_once "../php/libraries/Database.class.inc";
+require_once "../php/libraries/NDB_Config.class.inc";
+$path    = __DIR__ . "/../SQL/Release_patches";
+
+$files = array_diff(scandir($path), array('.', '..'));
```

Comment:
```suggestion
$files = glob("*.sql")
```

I think this is a bit more straightforward

Reply:
In this way, I can get file names with the path. I have to split the $files again.

Reply:
I only need the file names, in your way, I can get file names and path. I have to split the array again.

Reply:
You append the path in 43 though so you do need the path.

If you have the path you can always use `basename()` to get only the name.